diff --git a/dist/runtime-core.cjs.js b/dist/runtime-core.cjs.js
index 08899205dfe52109b460e9b8fc9847eb5b3241f7..a2b2377958ae2efd4f8e61a592aa44121f9e3db9 100644
--- a/dist/runtime-core.cjs.js
+++ b/dist/runtime-core.cjs.js
@@ -8726,6 +8726,7 @@ const setDevtoolsHook = setDevtoolsHook$1 ;
 const _ssrUtils = {
   createComponentInstance,
   setupComponent,
+  unsetCurrentInstance,
   renderComponentRoot,
   setCurrentRenderingInstance,
   isVNode: isVNode,
diff --git a/dist/runtime-core.cjs.prod.js b/dist/runtime-core.cjs.prod.js
index 8b2db9c0e26d3d18bf71bbf0922b61c555018708..b1e6a4021aa23fd96d17bd1c2585f27a3f734412 100644
--- a/dist/runtime-core.cjs.prod.js
+++ b/dist/runtime-core.cjs.prod.js
@@ -6818,6 +6818,7 @@ const setDevtoolsHook = shared.NOOP;
 const _ssrUtils = {
   createComponentInstance,
   setupComponent,
+  unsetCurrentInstance,
   renderComponentRoot,
   setCurrentRenderingInstance,
   isVNode: isVNode,
diff --git a/dist/runtime-core.esm-bundler.js b/dist/runtime-core.esm-bundler.js
index f0905f8afe50421feed03b208b19afb26e4bbb73..84df908bfed74a45ec109e315b5ce39c09383b5e 100644
--- a/dist/runtime-core.esm-bundler.js
+++ b/dist/runtime-core.esm-bundler.js
@@ -8801,6 +8801,7 @@ const setDevtoolsHook = !!(process.env.NODE_ENV !== "production") || true ? setD
 const _ssrUtils = {
   createComponentInstance,
   setupComponent,
+  unsetCurrentInstance,
   renderComponentRoot,
   setCurrentRenderingInstance,
   isVNode: isVNode,
