diff --git a/dist/server-renderer.cjs.js b/dist/server-renderer.cjs.js
index 3f485c1b55c58553a35fd1f3575d08dff549ea00..910fee845881c9d8405f84017ca12378e780586c 100644
--- a/dist/server-renderer.cjs.js
+++ b/dist/server-renderer.cjs.js
@@ -385,6 +385,7 @@ const {
   createComponentInstance,
   setCurrentRenderingInstance,
   setupComponent,
+  unsetCurrentInstance,
   renderComponentRoot,
   normalizeVNode,
   pushWarningContext,
@@ -435,7 +436,10 @@ function renderComponentVNode(vnode, parentComponent = null, slotScopeId) {
         );
       }
     }).catch(shared.NOOP);
-    return p.then(() => renderComponentSubTree(instance, slotScopeId));
+    return p.then(() => {
+      unsetCurrentInstance();
+      return renderComponentSubTree(instance, slotScopeId);
+    });
   } else {
     try {
       return renderComponentSubTree(instance, slotScopeId);
diff --git a/dist/server-renderer.cjs.prod.js b/dist/server-renderer.cjs.prod.js
index 6d88169f5fa6513abc92f5616a2038a9c817d98c..a85620c58eb521d1a10d9e2c93b19d7c7c6a9dcf 100644
--- a/dist/server-renderer.cjs.prod.js
+++ b/dist/server-renderer.cjs.prod.js
@@ -372,6 +372,7 @@ const {
   createComponentInstance,
   setCurrentRenderingInstance,
   setupComponent,
+  unsetCurrentInstance,
   renderComponentRoot,
   normalizeVNode,
   pushWarningContext,
@@ -420,7 +421,10 @@ function renderComponentVNode(vnode, parentComponent = null, slotScopeId) {
         );
       }
     }).catch(shared.NOOP);
-    return p.then(() => renderComponentSubTree(instance, slotScopeId));
+    return p.then(() => {
+      unsetCurrentInstance();
+      return renderComponentSubTree(instance, slotScopeId);
+    });
   } else {
     try {
       return renderComponentSubTree(instance, slotScopeId);
diff --git a/dist/server-renderer.esm-bundler.js b/dist/server-renderer.esm-bundler.js
index 428c09a2f945f08a2cfba23e363fa5b123f86dcd..dfcee9fa548c6b371d01857f231aa42db370934a 100644
--- a/dist/server-renderer.esm-bundler.js
+++ b/dist/server-renderer.esm-bundler.js
@@ -302,6 +302,7 @@ const {
   createComponentInstance,
   setCurrentRenderingInstance,
   setupComponent,
+  unsetCurrentInstance,
   renderComponentRoot,
   normalizeVNode,
   pushWarningContext,
@@ -352,7 +353,10 @@ function renderComponentVNode(vnode, parentComponent = null, slotScopeId) {
         );
       }
     }).catch(NOOP);
-    return p.then(() => renderComponentSubTree(instance, slotScopeId));
+    return p.then(() => {
+      unsetCurrentInstance();
+      return renderComponentSubTree(instance, slotScopeId);
+    });
   } else {
     try {
       return renderComponentSubTree(instance, slotScopeId);
